doc/source/methods/CFRL.ipynb

```diff
@@ -557,7 +557,7 @@
    "name": "python",
```

Comment:
Minor, but please keep an eye and try not to commit notebooks that have changed locally but should not be part of the PR (this one and many others below).